### DISCUSSION TITLE
is there a 32 bit desktop (sometimes called i386) verion of aws cli?  (x86)

### DISCUSSION THREAD

**Discussion by user706** [aws/aws-cli#7944] [Q&A] [answered]
is there a 32 bit desktop (sometimes called i386) verion of aws cli?   (x86)

Checking here...
https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html

...I only see reference to
```
https://awscli.amazonaws.com/awscli-exe-linux-x86_64-version.number.zip
                                                !!!
                                            64 bit version
```

Background: stuff like the generation of the rasbperry-pi image currently needs a i386 debian docker image ([ref](https://github.com/RPi-Distro/pi-gen/blob/01d24ef22778337ed04cf9d6444b1be57b6a1e1a/build-docker.sh#L86)), and if I want to extend this with aws, I have a problem -- I cannot find the 32 bit version of aws cli.

**Comment by indrora** [comment, marked as answer]
x86/ia32 usage isn't currently supported explicitly, however the AWS CLI v2 is written in pure Python. Our binary distribution packages a complete Python environment, including all the requisite packages such as `cryptography`. 

Update: You're free to attempt to use the Source Distribution, with instructions given here: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-source-install.html
